models/object.py

```diff
@@ -20,16 +19,16 @@ class Mode(Enum):
 
 # Styles for different states of the textbox
 class TextBoxStyles(Enum):
```

Comment:
idk if there was a reason these got changed, but they made the background of everything black